### DISCUSSION TITLE
I wanna help

### DISCUSSION THREAD

**Discussion by diegopaucarv** [ccbogel/QualCoder#811] [General]
So, long story short, I want to contribute to the project and to:

a) Optimize the user experience by creating templates and methodological guidance for specific qualitative techniques and designs (be it QCA, grounded theory approaches, themeing, or just customer experience insight-finding). For example, by adding prompts, strategically facilitating or warning about using certain functions according to the different research designs, or simply adding automatically translated audios to explain the user about different coding methods and steps. We would need some experiments, though, and that thrills me.
b) Incorporate NLP and maybe NLU (so-called AI, now that the open source Llama2 model has been published) into the software

I am sociologist, and I have published some papers already. This is one, using the Classic Grounded Theory approach. https://groundedtheoryreview.com/2020/12/30/how-do-waste-picker-families-endure-resolving-pains-and-managing-support-systems-as-close-relationship-resourcing-a-cgt-with-readily-available-data/.

I know Python and Java. I know some NLP with Python This is my gist (https://gist.github.com/diegopaucarv). I am studying to become a programmer, and I know some data structures and GUI creation.

**Comment by ccbogel** [comment]
HI thank you. 

a. improved user experience and prompts or videos would be good too. just need to have a good plan of what you want to do first.

b. I don't have any experience with NLP so yes something here would be good.
